woocommerce/woocommerce-ios
# Simple Payments: Release tax break up

# Why

I just noticed that we never releases the tax break-up feature implemented by @selanthiraiyan  on his trial project.

# How

- Removes feature flag

- Removes code that calculated the aggregated tax rate & tax total.

- Updates Unit tests.

# Screenshots 

No tax | Tax Lines
---- | ----
<img width="434" alt="no-tax" src="https://user-images.githubusercontent.com/562080/157766120-8c7daa2d-5d6e-4cf1-9129-2e3c09f6bbb7.png"> | <img width="436" alt="multiple" src="https://user-images.githubusercontent.com/562080/157766126-934a91a5-8763-4b2e-af78-c01366a17b44.png">


# Testing Steps
- Launch the app with a store that has multiple tax lines configured.
- Start a new simple payments order.
- Enter an amount & enable taxes
- See the correct tax break up.
